External/SPMmodified/toolbox/cat12/cat_main.m

```diff
@@ -951,17 +951,15 @@
   clear def; 
 
   % EXPLOREASL HACK
-    CurrDir=pwd;
-    if exist('./mri','dir')
-        cd('mri');
-    end
-    if xASL_exist('nT1.nii')
-        tIM = xASL_io_Nifti2Im('nT1.nii');
-        tIM(isnan(tIM)) = 0;
-        xASL_io_SaveNifti('nT1.nii', 'nT1.nii', tIM, [], false);
-    end
-    cd(CurrDir);
-  
+  CurrDir=pwd;
```

Comment:
Yes, I like this. You could only make it a bit more readable and less bug-prone by defining `PathT1` and `PathNT1` rather than repeating the `fullfile()`.

Reply:
Good point. Fixed.